backend/src/core/middlewares/campaign.middleware.ts

```diff
@@ -103,13 +103,19 @@ const listCampaigns = async (
   res: Response,
   next: NextFunction
 ): Promise<Response | void> => {
-  const { offset, limit } = req.query
+  const { offset, limit, type, status, name, sort_by, order_by } = req.query
```

Comment:
I wonder if it's possible to type this somewhere (since the shape described here is supposed to be consistent with that described in `getCampaigns` at `frontend/src/services/campaign.service.ts`)

that way you get type hinting if you modify one part and forgot to change the other part

(for context, Postman began as a JavaScript codebase and only added TypeScript afterwards, so don't think the typing practice here are ideal)